{"uid":"e135492f0ee72e7e","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"d8696551571014a71a874c85cadaf3fe","time":{"start":1706490559352,"stop":1706490569626,"duration":10274},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556f492a3f83 <unknown>\n#1 0x556f48f5ccf7 <unknown>\n#2 0x556f48fac99e <unknown>\n#3 0x556f48facaa1 <unknown>\n#4 0x556f48ff7d64 <unknown>\n#5 0x556f48fd60dd <unknown>\n#6 0x556f48ff5006 <unknown>\n#7 0x556f48fd5e53 <unknown>\n#8 0x556f48f9ddd4 <unknown>\n#9 0x556f48f9f1de <unknown>\n#10 0x556f49268531 <unknown>\n#11 0x556f4926c455 <unknown>\n#12 0x556f49254f55 <unknown>\n#13 0x556f4926d0ef <unknown>\n#14 0x556f4923899f <unknown>\n#15 0x556f49291008 <unknown>\n#16 0x556f492911d7 <unknown>\n#17 0x556f492a3124 <unknown>\n#18 0x7fa800694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f84e06f8cb0>\nemail = 'nirt236@gmail.com', password = '123456'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"be7a756d76156549c268c89a8da3cedc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f84e05cd6c0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x556f492a3f83 <unknown>\nE       #1 0x556f48f5ccf7 <unknown>\nE       #2 0x556f48fac99e <unknown>\nE       #3 0x556f48facaa1 <unknown>\nE       #4 0x556f48ff7d64 <unknown>\nE       #5 0x556f48fd60dd <unknown>\nE       #6 0x556f48ff5006 <unknown>\nE       #7 0x556f48fd5e53 <unknown>\nE       #8 0x556f48f9ddd4 <unknown>\nE       #9 0x556f48f9f1de <unknown>\nE       #10 0x556f49268531 <unknown>\nE       #11 0x556f4926c455 <unknown>\nE       #12 0x556f49254f55 <unknown>\nE       #13 0x556f4926d0ef <unknown>\nE       #14 0x556f4923899f <unknown>\nE       #15 0x556f49291008 <unknown>\nE       #16 0x556f492911d7 <unknown>\nE       #17 0x556f492a3124 <unknown>\nE       #18 0x7fa800694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1706490526857,"stop":1706490526857,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1706490526857,"stop":1706490526857,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1706490526857,"stop":1706490526858,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556f492a3f83 <unknown>\n#1 0x556f48f5ccf7 <unknown>\n#2 0x556f48fac99e <unknown>\n#3 0x556f48facaa1 <unknown>\n#4 0x556f48ff7d64 <unknown>\n#5 0x556f48fd60dd <unknown>\n#6 0x556f48ff5006 <unknown>\n#7 0x556f48fd5e53 <unknown>\n#8 0x556f48f9ddd4 <unknown>\n#9 0x556f48f9f1de <unknown>\n#10 0x556f49268531 <unknown>\n#11 0x556f4926c455 <unknown>\n#12 0x556f49254f55 <unknown>\n#13 0x556f4926d0ef <unknown>\n#14 0x556f4923899f <unknown>\n#15 0x556f49291008 <unknown>\n#16 0x556f492911d7 <unknown>\n#17 0x556f492a3124 <unknown>\n#18 0x7fa800694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f84e06f8cb0>\nemail = 'nirt236@gmail.com', password = '123456'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"be7a756d76156549c268c89a8da3cedc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f84e05cd6c0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x556f492a3f83 <unknown>\nE       #1 0x556f48f5ccf7 <unknown>\nE       #2 0x556f48fac99e <unknown>\nE       #3 0x556f48facaa1 <unknown>\nE       #4 0x556f48ff7d64 <unknown>\nE       #5 0x556f48fd60dd <unknown>\nE       #6 0x556f48ff5006 <unknown>\nE       #7 0x556f48fd5e53 <unknown>\nE       #8 0x556f48f9ddd4 <unknown>\nE       #9 0x556f48f9f1de <unknown>\nE       #10 0x556f49268531 <unknown>\nE       #11 0x556f4926c455 <unknown>\nE       #12 0x556f49254f55 <unknown>\nE       #13 0x556f4926d0ef <unknown>\nE       #14 0x556f4923899f <unknown>\nE       #15 0x556f49291008 <unknown>\nE       #16 0x556f492911d7 <unknown>\nE       #17 0x556f492a3124 <unknown>\nE       #18 0x7fa800694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1706490559352,"stop":1706490569625,"duration":10273},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556f492a3f83 <unknown>\n#1 0x556f48f5ccf7 <unknown>\n#2 0x556f48fac99e <unknown>\n#3 0x556f48facaa1 <unknown>\n#4 0x556f48ff7d64 <unknown>\n#5 0x556f48fd60dd <unknown>\n#6 0x556f48ff5006 <unknown>\n#7 0x556f48fd5e53 <unknown>\n#8 0x556f48f9ddd4 <unknown>\n#9 0x556f48f9f1de <unknown>\n#10 0x556f49268531 <unknown>\n#11 0x556f4926c455 <unknown>\n#12 0x556f49254f55 <unknown>\n#13 0x556f4926d0ef <unknown>\n#14 0x556f4923899f <unknown>\n#15 0x556f49291008 <unknown>\n#16 0x556f492911d7 <unknown>\n#17 0x556f492a3124 <unknown>\n#18 0x7fa800694ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"4d10b56fd3f30bc4","name":"public ip address","source":"4d10b56fd3f30bc4.txt","type":"text/plain","size":12},{"uid":"e5b1157e165a3af2","name":"Cookies","source":"e5b1157e165a3af2.json","type":"application/json","size":3872},{"uid":"ee6862530a4deb15","name":"Session Storage","source":"ee6862530a4deb15.json","type":"application/json","size":359},{"uid":"f27bb19c92a96249","name":"Local Storage","source":"f27bb19c92a96249.json","type":"application/json","size":208},{"uid":"88de560677734e07","name":"Console Logs","source":"88de560677734e07.json","type":"application/json","size":2},{"uid":"e3c135d4d0631715","name":"Full Page Screenshot","source":"e3c135d4d0631715.png","type":"image/png","size":5702497},{"uid":"bba646f6deb9843","name":"URL","source":"bba646f6deb9843.uri","type":"text/uri-list","size":23},{"uid":"6e55f88758b4472a","name":"Network Logs","source":"6e55f88758b4472a.json","type":"application/json","size":23160}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1382-482"},{"name":"thread","value":"2453-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'nirt236@gmail.com'"},{"name":"password","value":"'123456'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"e135492f0ee72e7e.json","parameterValues":["'nirt236@gmail.com'","'123456'"]}